### PR TITLE
Create PR to merge version commit into working release branches

### PR DESCRIPTION
**Pull request by djgalloway**

### CODE REVIEW

File: ansible/roles/ceph-release/tasks/push.yml [hunk 1]
```diff
@@ -41,21 +41,22 @@
...
+  uri:
+    url: https://api.github.com/repos/ceph/ceph/pulls
+    method: POST
+    status_code: 201
+    headers:
+      Accept: "application/vnd.github.v3+json"
+      Authorization: "token {{ token }}"
+    body:
+      title: "v{{ version }}"
+      body: "{{ pr_checklist }}"
```
Comment (batrick) [inline ansible/roles/ceph-release/tasks/push.yml:54]: It'd be better to just read the file `.github/pull_request_template.md` before this task, no?

**Review by batrick — CHANGES_REQUESTED** [review]

File: ansible/roles/ceph-release/tasks/push.yml (hunk 1 above)
Comment (djgalloway) [reply ansible/roles/ceph-release/tasks/push.yml:54]: Read it and then programmatically check the boxes? Is that what you're suggesting? I suppose.

Comment (djgalloway) [reply ansible/roles/ceph-release/tasks/push.yml:54]: Actually, even if the checklist changes, I'd still have to go in this playbook and change which boxes get checked so it's kind of a moot point.

Comment (djgalloway) [reply ansible/roles/ceph-release/tasks/push.yml:54]: @batrick can we merge this as-is now?

Comment (batrick) [reply ansible/roles/ceph-release/tasks/push.yml:54]: ya

**Review by batrick — APPROVED** [review]

**Pull request merged**